### PR TITLE
refactor: Declare fields when fetching orgas and use the proper rels

### PR DESCRIPTION
**Pull request by salisdemos**
- we don't want to access the customer directly from the orga. Its deprecated
- and we should define explicitly which fields we want to get from the Backend
- even if that means listing all fields.

As Mandaten-Admin / Support The Page `/organisation/list` should work like before but without the purple warnings

### CODE REVIEW

File: client/js/bundles/user/listOrgas.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const presetStoreModules = {
           }
         }
       },
-      include: ['branding', 'currentSlug', 'customers', 'statusInCustomers'].join(),
+      include: ['customer.branding', 'currentSlug', 'customers', 'statusInCustomers'].join(),
```
Comment (salisdemos) [inline client/js/bundles/user/listOrgas.js:41]:
```suggestion
      include: ['customer.branding', 'currentSlug', 'statusInCustomers'].join(),
```

**Review by martendemos — APPROVED** [review]
BE: lgtm

**Review by dresslerdemos — APPROVED** [review]
backend review only

**Review by elmasdemos — APPROVED** [review]
code review, lgtm and it works. thanks

**Pull request merged**